### PR TITLE
ci: update nodejs versions

### PR DESCRIPTION
**Pull request by ivan-tymoshenko**

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x]
```
Comment (mcollina) [inline .github/workflows/node.js.yml:15]: We need to keep supporting Node v14 for supporting Fastify v4. Just exclude Windows v14 which is broken on GHA.

**Review by mcollina — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (ivan-tymoshenko) [reply .github/workflows/node.js.yml:15]: we still suuport v14? 🤦🏻

Comment (mcollina) [reply .github/workflows/node.js.yml]: we need to plan fastify v5

**Review by mcollina — APPROVED** [review]
lgtm

**Pull request merged**